### PR TITLE
H8462 improve ESP tracing

### PR DESCRIPTION
**Pull request by wangkx**
Add 'GET/POST/SOAP service.method version' to TxSummary;
Set log request to be default;
Clean extra log lines;
Add log for starting WU schedule threads;
Log important ECLWatch service calls;
Log QueryFilesInUse subscription/notification.

### CODE REVIEW

**Comment by wangkx** [comment]
@afishbeck please review.

**Comment by richardkchapman** [comment]
Needs rebasing for a clean merge

**Comment by wangkx** [comment]
Rebased to the latest candidate-6.0.0.

File: esp/services/ws_fs/ws_fsService.cpp [hunk 1]
```diff
@@ -1664,8 +1672,12 @@ bool CFileSprayEx::onDeleteDFUWorkunits(IEspContext &context, IEspDeleteDFUWorku
         StringArray & wuids = req.getWuids();
         for(unsigned i = 0; i < wuids.ordinality(); ++i)
         {
-            if (markWUFailed(factory, wuids.item(i)))
-                factory->deleteWorkUnit(wuids.item(i));
+            const char* wuid = wuids.item(i);
+            if (markWUFailed(factory, wuid))
+            {
+                factory->deleteWorkUnit(wuid);
+                PROGLOG("%s deleted", wuid);
```
Comment (afishbeck) [inline esp/services/ws_fs/ws_fsService.cpp]: Might want to be a bit more descriptive on these deletes... so we can easily find the line that did the delete..

 PROGLOG("DeleteDFUWorkunits: %s deleted", wuid);?

File: esp/services/ws_fs/ws_fsService.cpp [hunk 2]
```diff
@@ -1684,9 +1696,13 @@ bool CFileSprayEx::onDeleteDFUWorkunit(IEspContext &context, IEspDeleteDFUWorkun
...
             throw MakeStringException(ECLWATCH_DFU_WU_ACCESS_DENIED, "Failed to delete DFU workunit. Permission denied.");
 
+        const char* wuid = req.getWuid();
         Owned<IDFUWorkUnitFactory> factory = getDFUWorkUnitFactory();
-        if (markWUFailed(factory, req.getWuid()))
-            resp.setResult(factory->deleteWorkUnit(req.getWuid()));
+        if (markWUFailed(factory, wuid))
+        {
+            resp.setResult(factory->deleteWorkUnit(wuid));
+            PROGLOG("%s deleted", wuid);
```
Comment (afishbeck) [inline esp/services/ws_fs/ws_fsService.cpp]: PROGLOG("DeleteDFUWorkunit: %s deleted", wuid);?

**Comment by afishbeck** [comment]
@wangkx looks fine

**Comment by wangkx** [comment]
@richardkchapman this PR is ready to be merged.

**Pull request merged**